assembl/static2/js/app/components/form/datePickerFieldAdapter.jsx

```diff
@@ -0,0 +1,97 @@
+// @flow
+
+/*
+  DatePicker adapter for react-final-form that supports the following form:
+  { time: moment().utc() }
+
+  Errors are managed by a Mutator on the Form itself. As a result, to show errors
+  on DatePickerFieldAdapter, must add the `setFieldTouched` from final-form-set-field-touched
+  along with passing the form <FormApi> prop
+*/
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import * as React from 'react';
+import { ControlLabel, FormGroup } from 'react-bootstrap';
+import { type FieldRenderProps } from 'react-final-form';
+import { type FormApi } from 'final-form';
+import type { DatePickerType, DateTime, DatePickerValue } from './types.flow';
+import Error from './error';
+import { getValidationState } from './utils';
+
+type Props = {
+  editLocale: string,
+  picker?: DatePickerType,
+  placeHolder: string,
+  showTime: boolean,
+  hasConflictingDate: boolean,
+  input: {
+    name: string,
+    onChange: (SyntheticInputEvent<*> | any) => void,
```

Comment:
`SyntheticInputEvent<*> | any` is the same as `any`. Could you have avoided the `|any` here?

Reply:
I changed the flow annotation in https://github.com/assembl/assembl/commit/bc037b77900c91216796bd0bbb171a601970e8df